std/wasi/snapshot_preview1.ts

```diff
@@ -228,7 +228,8 @@ const clock_time_monotonic = function (): bigint {
 const clock_time_process = clock_time_monotonic;
 const clock_time_thread = clock_time_monotonic;
 
-function syscall(target: Function): Function {
+type SyscallFunction = (...args: unknown[]) => number;
+function syscall(target: SyscallFunction): SyscallFunction {
```

Comment:
Actually, on second thought I think you have to use a generic type here and forward it.

Reply:
Thanks again, I will try to fix.

Reply:
I tried to avoid using `any` type, but I didn't figure out the way. But I think it should pass the CI...